tests/e2e/koperator_suite_test.go

```diff
@@ -54,12 +54,12 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = When("Testing e2e test altogether", Ordered, func() {
-	//testInstall()
+	testInstall()
 	testInstallZookeeperCluster()
 	testInstallKafkaCluster("../../config/samples/simplekafkacluster.yaml")
 	testUninstallKafkaCluster()
 	testInstallKafkaCluster("../../config/samples/simplekafkacluster_ssl.yaml")
 	testUninstallKafkaCluster()
 	testUninstallZookeeperCluster()
-	//testUninstall()
+	testUninstall()
```

Comment:
**Question**: Unrelated to the PR, but why are the producer/consumer tests not enabled here BTW?

Reply:
No idea, probably whoever merged the last PR (Might have been me) didn't work on those and overwrote this file.

Reply:
I see it will be fixed in #1017.